account/lnurl.go

```diff
@@ -421,18 +427,6 @@ func (a *Service) FinishLNURLPay(params *data.LNURLPayResponse1) (*data.LNUrlPay
 		return nil, errors.New("Invoice amount does not match the amount set by user.")
 	}
 
-	/* TODO 9. If routes array is not empty: verifies signature for every provided `ChannelUpdate`, may use these routes if fee levels are acceptable.
-
-	   ref. https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#the-channel_update-message
-	   ref. github.com\lightningnetwork\lnd\lnwire\channel_update.go
-
-	   ref. github.com/lightningnetwork/lnd/lnwire/channel_update.go
-	   ref. github.com/lightningnetwork/lnd/lnwire/message.go
-	*/
-	if len(payResponse2.Routes) > 0 {
```

Comment:
go-lnurl has changed Routes to `interface{}` so IMHO this code does not make sense anymore